src/api/helpers/ApiConstant.ts

```diff
@@ -0,0 +1,24 @@
+export const MAIN_API_URLS = {
+  BASE: 'https://back-test-4zwpv.ondigitalocean.app/api',
```

Comment:
Sometimes better if we use .env file for containing some data about urls/tokens. For example, if url have different values for dev and prod versions, better if we use it from .env file.
So, in this place you can use something like this: 
`process.env.REACT_APP_API_BASE_URL || 'https://back-test-4zwpv.ondigitalocean.app/api',`.

Reply:
Fix this